### ISSUE TITLE
failing to load flipnote

### ISSUE THREAD

**Issue by Ponali** [jaames/flipnote-player#30]
i'm trying to load a flipnote to the website, however everytime i try to, the error "This Flipnote could not be loaded." error appears.
i may need more context on what went wrong if i want to fix it myself.

file: [file.zip](https://github.com/jaames/flipnote-player/files/14941542/file.zip)

**Comment by jaames** [comment]
Are you trying to encode a Flipnote using a PC tool or something? I only skimmed through it in a hex editor and looked at some important parts, but the file you provided doesn't look quite right to me.

**Comment by Ponali** [comment]
yeah, i'm actually trying to make a node.js script that generates a flipnote from some math. for the moment, there is no compression for lines, only type 3, so that's why the flipnote is very big
i'm trying to make this flipnote encoder so i can try porting flipnote studio to computers, and adding more functionality like making it use paint-like usage for making them.

i checked the file myself on a hex editor, and [a flipnote from radxflipnote](https://www.sudomemo.net/watch/DC65E9_16B981458F189_000), and i never found anything that was wrong.

i reccomend you continue the discussion by opening a new issue in [this repo](https://github.com/Ponali/flipnode).

**Comment by jaames** [comment]
This probably isn't the reply you want, so I'm sorry in advance, but if your project is not producing a Flipnote that can be played on this website, then I'm kind of doubtful that your files can be played in Flipnote Studio too. The underlaying PPM parser here is more or less a line-for-line translation of the original Flipnote Studio implementation. 

So that leaves the most likely scenario as being that there's a bug in your code. Unfortunately I don't really have time at the moment to help you with that. It's kind of up to you to dig a bit deeper.

Obviously there could be something that I've missed! If you have a file that plays inside of Flipnote Studio but doesn't play in this website, by all means reopen this issue.

Cheers.
